reading.Rmd

```diff
@@ -169,8 +174,11 @@ We put `data/` before the file's
 name when we are loading the data set because this data set is located in a
 sub-folder, named `data`, relative to where we are running our R code.
 
-Here is what the file would look like in a plain text editor (a program that removes
-all formatting, like bolding or different fonts):
+Here is what the file would look like in a plain text editor.
+
+> Note: A plain text editor is a program that that allows for editing of plain text. 
+Plain text is text represented only by characters 
```

Comment:
I found this note box to be a bit abstract/unclear. What is "meta-specification data", "character specification data", "paragraph specification data", etc ?

Reply:
That is the formatting data from rich text. We can change how this is worded, but we cannot simply revert back to what we had before because it was wrong (plain text editors do no "remove" formatting from text/rich text. They just represent everything as characters.